corehq/apps/app_manager/templates/app_manager/partials/forms/case_config_ko_templates.html

```diff
@@ -106,7 +106,7 @@
                       casePropertyAutocomplete: suggestedProperties"/>
     </span>
   <span data-bind="visible: property.required()">
-    <code data-bind="text: property.key"></code>
```

Comment:
It's possible for there to be other required properties, so it isn't safe to hardcode `case_name` here. This particular change looks like it's going to be more involved. [Here](https://github.com/dimagi/commcare-hq/blob/0a5391e1b5c7ae2c68d28d15a420fb791050cb98/corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js#L271) is where `name` is added as a required property. You'd need to add an additional attribute to the `casePropertyBase` class in that file to contain the user-facing name.

That said, I'm surprised that we're going with `case_name` rather than `name`. I just made a comment on the spec to ask for more info about this, so there's no need to sink a bunch of time into changing this code this until that question is resolved.

Reply:
Looked into this some more. I had been thinking `name` is the "real" name, but I was wrong about that. So I'd start by updating that line in `case_config_ui.js` to use `case_name` instead of `name`, and likely a few other places in that file that use `name`, like [here](https://github.com/dimagi/commcare-hq/blob/5f3ba9fca3e558fff162a885b5a3af84a17f8ffb/corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js#L313-L315).

The [advanced case config UI](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_config_ui.js) should be updated similarly. To test that code, you'll need to turn on the advanced modules feature flag and add an advanced module to your test app.